docs/examples/integration-test.tsx

```
@@ -0,0 +1,239 @@
+            placeholder="Name"
+            data-testid="name-input"
+          />
+          <button type="submit" data-testid="submit-button">Add User</button>
+        </form>
+      </div>
+    );
+  }
+}));
+
+// The component under test
+const UserManagement = () => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'UserManagement'. Consider to delete it or rename it.

'UserManagement' is defined here:


(lint/suspicious/noRedeclare)